### PR TITLE
pushing pictures for 2_First_Steps_in_r

### PR DESCRIPTION
**Pull request by jonathansoule**

### CODE REVIEW

File: WorkingInR/0_Preamble.Rmd [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+# Preamble
+
+## About this website
+
+These pages are the result of a collaborative effort between bioCEED, Centre for Excellence in Biology Education, the Department of Biological Sciences at the University of Bergen, and the Department of Biology and the Department of Mathematical Sciences at the Norwegian University of Science and Technology.
+
+## Who is it for?
+
```
Comment (audhalbritter) [inline WorkingInR/0_Preamble.Rmd:23]: Mainly biology students at bachelor and master level. We also have more advanced stuff for higher levels.
Some of the stuff links directly up to BIO courses at Uib and NTNU.

File: WorkingInR/0_Preamble.Rmd [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ 
+```{r app0, echo = FALSE, eval = TRUE, fig.cap="_Screenshot of an interactive app._", out.width="100%"}
+knitr::include_graphics("Pics/app0.png")
+```
+
+Sometimes, the app will be pre-filled with the full code ready to run. 
+Sometimes the code will be partial or incorrect, meaning that you will have to edit it first. 
+In all cases, run the code by clicking the green button `Run Code` in the upper right corner. 
+Alternatively, start again from the beginning by clicking the `Start Over` button.
+
```
Comment (audhalbritter) [inline WorkingInR/0_Preamble.Rmd:56]: ### Biostats R package

For more in depth info about specific topics we have an r package
This is how to install it and load it...

File: WorkingInR/0_Preamble.Rmd [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+### Before we start
+
+At the top of each page, you will find the `Before we start` box which tells you what you must already know or be familiar with before you read further. 
+
+### Body
+
+The body of the page introduces the topic(s) along with relevant examples.
+
+### Interactive apps
```
Comment (audhalbritter) [inline WorkingInR/0_Preamble.Rmd:42]: It's interactive Learner sections... not sure what to call it.

File: WorkingInR/0_Preamble.Rmd [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+R is a language; thus it is **important to practice writing** to learn it better and faster. 
+To do so, we have included a set of interactive apps which you may use to try the code. 
+Figure \@ref(fig:app0) shows how such an interactive app looks like.
+ 
+```{r app0, echo = FALSE, eval = TRUE, fig.cap="_Screenshot of an interactive app._", out.width="100%"}
+knitr::include_graphics("Pics/app0.png")
+```
+
+Sometimes, the app will be pre-filled with the full code ready to run.
```
Comment (audhalbritter) [inline WorkingInR/0_Preamble.Rmd:52]: Maybe also say that sometimes there are tips, and you can see the solution by clicking on the solution button.

File: WorkingInR/0_Preamble.Rmd (hunk 1 above)
Comment (jonathansoule) [reply WorkingInR/0_Preamble.Rmd:23]: Yes. suggestion:

These pages are essentially meant for biology students both at the bachelor and master level, and link up to courses held at UiB and NTNU.
Contents may certainly be useful for other educational programmes (biomedicine, etc) and other institutions. 
Some of our pages offer more advanced content which may be relevant at doctoral levels and in research generally.

File: WorkingInR/0_Preamble.Rmd (hunk 3 above)
Comment (jonathansoule) [reply WorkingInR/0_Preamble.Rmd:42]: What about "Interactive Learnr tutorials"

File: WorkingInR/0_Preamble.Rmd (hunk 4 above)
Comment (jonathansoule) [reply WorkingInR/0_Preamble.Rmd:52]: I've added a few minor changes to the first part of the text, and the following to the end:

Some of these applications are set up as more or less complex exercises. 
Two additional buttons may be available: `Hint` and `Solution`. 
`Hint` may provide you with fragments of the code or tips that will point you in the right direction; `Solution` reveals the answer.

File: WorkingInR/0_Preamble.Rmd (hunk 2 above)
Comment (jonathansoule) [reply WorkingInR/0_Preamble.Rmd:56]: we can discuss about how to present that package (specific section, or just an additional paragraph)

**Pull request closed without merging**